New readme.

coderanger opened the pull request:
Ping @mahmoudimus @bninja @mjallday

coderanger committed:
New readme.

mahmoudimus commented:
how to pack?

mahmoudimus commented:
how to use cloudformation w/ brix w/o using strataosphere?

bninja commented on README.md:
```diff
@@ -1,34 +1,84 @@
 Adding A Template
 -----------------
 
 To add a new template you need to:
```
doesnt render as list for me

coderanger committed:
Try to fix list formatting.

coderanger committed:
Add instructions for Packer.

For posterity the full command I use is:

env AWS_ACCESS_KEY_ID=$BALANCED_AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY=$BALANCED_AWS_SECRET_ACCESS_KEY packer build balanced-client.json

coderanger committed:
Typo.

coderanger commented:
Docs added for packer, will hold off on the non-stratosphere docs until someone actually tries it :-)
